_includes/static/navbar.html

```diff
@@ -4,10 +4,9 @@
     <div class="col">
       <div class="row justify-content-start">
         <div class="col-sm-4">
-          <a href="{{ site.baseurl }}">
+          <a href="../../">
```

Comment:
```suggestion
          <a href="/">
```
I think we can use a single `/` for this. Try it and let us know

Reply:
Done